pkg/controller/master/image/vm_image_controller.go

```diff
@@ -111,7 +113,12 @@ func (h *vmImageHandler) initialize(image *harvesterv1.VirtualMachineImage) (*ha
 		h.imageController.EnqueueAfter(image.Namespace, image.Name, 10*time.Second)
 		return h.images.Update(image)
 	}
-	return h.createBackingImageAndStorageClass(image)
+
+	image, err = h.createBackingImageAndStorageClass(image)
```

Comment:
Both `checkImage` and `createBackingImageAndStorageClass` call `toUpdate := image.DeepCopy()`. How about we do it in `initialize` and send the copied one to these two functions?

Reply:
Sure, addressed in  https://github.com/harvester/harvester/pull/5498/commits/3156aa6a01658b01bd673be6e429e2fce9733d4b